### DISCUSSION TITLE
add ability to rename columns and filter relational data in table layout

### DISCUSSION THREAD

**Discussion by wc-matteo** [directus/directus#12269] [Feature Requests]
The [latest](https://github.com/directus/directus/pull/12154) upgrades to table layout are really cool! Now if only we could rename columns and filter relational data (right now multiple values are concatenated by a comma) for the layout...

**Comment by wc-matteo** [comment]
one of the most common use cases for filtering, I think, would be translations. You want to show a specific language for the relational columns

**Reply by rijkvanzanten** [reply]
Absolutely. This is actually the same discussion as on https://github.com/directus/directus/issues/12271, so I'd love to hear your thoughts there too.

> Now if only we could rename columns

Do you mean rename them on the layout itself, or globally? You can already add custom translations for the names of collections/fields, which will then be used all over the app (including on the table layout) 🙂

**Reply by wc-matteo** [reply]
> Do you mean rename them on the layout itself, or globally

yeah on the layout. I use the field names translations all the time. But sometimes a particular name makes sense only for the edit item view, or only for the items list view (e.g. table layout)

**Reply by wc-matteo** [reply]
> sometimes a particular name makes sense only for the edit item view

an example would be the translations relational field. In the edit item view I usually have it as: "Languages" but in the table layout it doesn't make much sense, since I use the (recently added) translations display. So in that case the translations field would act more like a preview of the item's contents. Although now that we can add relational data to the table layout, that wouldn't be needed anymore. But I'm sure it would still be useful to change the columns names at the preset level

**Reply by rijkvanzanten** [reply]
Gotcha! Thanks for clarifying that 👍🏻

**Comment by arladmin** [comment]
Is there any workaround for this currently, if not a permanent solution?

This is getting a bit out of hand: 😐

![image](https://user-images.githubusercontent.com/64059154/223162857-a12c8b4e-2942-468d-a201-e7ec810bbc5f.png)

**Reply by rasco** [reply]
Yes I have exactly the same case, but with two columns named "Title".
It has a big negative usability impact.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
